gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/query_service/api/ApiEventQueryServiceImpl.java

```diff
@@ -71,4 +67,13 @@ public Optional<Api> findLastPublishedApi(String organizationId, String environm
             }
         });
     }
+
+    private static EventCriteria buildCriteria(String environmentId, String apiId) {
```

Comment:
I usually implement a unit test using mock for that kind of service. It allows us to ensure Critieria are correctly built.

Reply:
@jgiovaresco test is pushed